target/linux/airoha/an7581/target.mk

```diff
@@ -2,6 +2,7 @@ ARCH:=aarch64
 SUBTARGET:=an7581
 BOARDNAME:=AN7581 / AN7566 / AN7551
 CPU_TYPE:=cortex-a53
+DEFAULT_PACKAGES += wpad-basic-mbedtls uboot-envtools airoha-en7581-npu-firmware
```

Comment:
Considering that not every AN7581 device has Wi-Fi, it would
be more appropriate to put wpad in DEVICE_PACKAGES.